script/tool/lib/src/gradle_check_command.dart

```diff
@@ -327,12 +327,16 @@ plugins {
   /// compatibility with apps that use AGP 8+.
   bool _validateNamespace(RepositoryPackage package, String gradleContents,
       {required bool isExample}) {
-    final RegExp namespaceRegex =
+    final RegExp namespaceBlockRegex =
         RegExp('^\\s*namespace\\s+[\'"](.*?)[\'"]', multiLine: true);
```

Comment:
Couldn't we just update this regex to `'^\\s*namespace\\s+=?\\s*[\'"](.*?)[\'"]'` instead of adding a second regex?

Reply:
Also I find common examples above regex helpful for understanding could you add the 2 main examples we are trying to match.